metrics.go

```diff
@@ -0,0 +1,86 @@
+package es
+
+import (
+	"net/http"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/elastic/go-elasticsearch/v7/estransport"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var clientDurationSummary *prometheus.SummaryVec
+
+func init() {
+	clientDurationSummary = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Name: "es_search_operations_durations_seconds",
+			Help: "es_search_operations_durations_seconds",
```

Comment:
as far as we are wrapping transport - we dont need `search` in the metric name. so it could be just `es_operation_duration_seconds`